src/test/ui/feature-gate-isa_attribute.rs

```diff
@@ -0,0 +1,6 @@
+// only-armv4t-unknown-linux-gnueabi
```

Comment:
@jonas-schievink this doesn't work either, it's running on intel architecture tests in CI :confused:

Reply:
It does not, all `ui` tests pass or are ignored: `test result: ok. 10711 passed; 0 failed; 75 ignored; 0 measured; 0 filtered out`.

Reply:
oh I guess I'm just thrown by the mass of information from the CI or had an old run open. Let me have another look and try to figure out what the current issue is